src/plugins/vis_type_timeseries/public/actions/last_value_mode_action.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EmbeddableInput, EmbeddableOutput, IEmbeddable } from '../../../embeddable/public';
+import { createAction } from '../../../ui_actions/public';
+import { TIME_RANGE_DATA_MODES } from '../../common/timerange_data_modes';
+import { PANEL_TYPES } from '../../common/panel_types';
+import { Vis } from '../../../visualizations/public';
+
+const LAST_VALUE_MODE = 'lastValueMode';
```

Comment:
Why do not you use constants from here `src\plugins\vis_type_timeseries\common\timerange_data_modes.ts`?